src/MetadataFactory.php

```diff
@@ -30,11 +30,6 @@ class MetadataFactory implements AdvancedMetadataFactoryInterface
      */
     private $loadedClassMetadata = [];
 
-    /**
-     * @var null|string
-     */
-    private $hierarchyMetadataClass;
```

Comment:
I guess this property should not have been removed

Reply:
I remove this variable because the coding style check will be failed.

Here is the [history log](https://travis-ci.org/open-source-contributions/metadata/jobs/602190788).

Reply:
it is the bug of the style checker, the property is being read at https://github.com/schmittjoh/metadata/blob/8eec675b23046f096373252002baf54653939886/src/MetadataFactory.php#L152

Reply:
Ok. Reverting this change :).